src/volunteers.html

```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
 <section>
   <div class="container">
     <h2 class="section-heading">How it works</h2>
-    <div class="col-wrap">
-      <div class="col-3 centered-text">
+    <div class="container">
+      <div class="col-md-4 centered-text" align="center">
         <img class="non-profit-icons" src="img/icons/email-us.svg" alt="Email Us Image">
         <h2 class="section-heading">1. Contact Us</h2>
         <p>Fill in the <a class="text-link" href="volunteer-contact.html">volunteer form</a> and hit send.</p>
       </div>
-      <div class="col-3">
+      <div class="col-md-4" align="center">
         <img class="non-profit-icons" src="img/icons/email-confirmation.svg" alt="Email Confirmation Image">
         <h2 class="section-heading">2. Confirmation</h2>
         <p>We will email you all the info you need to get started.</p>
       </div>
-      <div class="col-3">
+      <div class="col-md-4" align="center">
         <img class="non-profit-icons" src="img/icons/project-starts.svg" alt="Email Confirmation Image">
         <h2 class="section-heading">3. Done!</h2>
         <p>Choose your project and get hacking!</p>
+        </div>
       </div>
     </div>
-  </div>
+
 </section>
 <section>
   <h2 class="section-heading">Meet our volunteers</h2>
   <p>Meet some of our volunteers to find out what you could be both contributing and learning by joining WHFNP.</p>
   <div class="volunteers-wrap">
-    <div class="col-2">
+  <div class="row">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
       <a href="http://marizadima.wordpress.com" class="volunteer-link">
         <img class="volunteer-picture" src="img/volunteers/Mariza_Dima.png" alt="Mariza Dima Picture">
         <h2>Mariza Dima</h2>
       </a>
       <p class="volunteer-quote">"It's a rewarding experience to work as a tech developer for good causes and even more so in collaboration with women. In WHFNP we bring together strong expertise and project management with a feminine perspective in providing solutions to complex problems, and designing applications that can make a change."</p>
     </div>
-    <div class="col-2">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
```

Comment:
Why do we need `col-lg-offset-0`? That does an offset of 0 columns, can it be removed unless it does something special?